README.md

```diff
@@ -143,20 +146,21 @@ You can access your app's translations in templates using the `t` helper:
 Pass values to be interpolated into the translation as [hash arguments](http://handlebarsjs.com/expressions.html). For example, for a translation that includes an interpolated `{{count}}` value:
 
 ```handlebars
-<div>{{t 'messages.count' count=3}}</div>
+<div>{{t 'messages.count' count=model.messageCount}}</div>
 ```
 
 ### Current Locale
 
-The current locale is exposed via a `locale` property added to the i18n service. To change the language that the appliction is displayed in, simply set this property, and all of the text displayed using the `t` helper will be updated. For example, triggering the following controller action would update all of the text to Thai:
+The current locale is exposed via i18n service's `locale` property. To change the language that the application is displayed in, simply set this property, and all of the text displayed using the `t` helper will be updated. For example, triggering the following controller action would update all of the text to Thai:
 
 ```javascript
 // app/controllers/example-controller.js
-import Ember from 'ember';
+import Controller from '@ember/controller';
+import I18nMixin from 'ember-i18next/mixins/i18n';
 
-export default Ember.Controller.extend(I18nMixin, {
+export default Controller.extend(I18nMixin, {
   actions: {
-    gimmeThai() {
+    showThai() {
       this.set('i18n.locale', 'th-TH');
```

Comment:
This usage of the Thai locale uses uppercase, but in a later example it is all lowercase. We should probably be consistent to avoid confusion.

Reply:
Good point. I'll update all the locales so they're consistent.